torch/csrc/jit/python/script_init.cpp

```diff
@@ -1303,11 +1305,21 @@ void initJitScriptBindings(PyObject* module) {
                    "something else!";
           }
           methodDefs.emplace_back(Def(def));
-          rcbs.push_back(
+          methodRcbs.push_back(
               pythonResolver(rcb, classDef.name().name(), classType));
         }
+
+        // Compile getters and setters for properties as regular methods.
```

Comment:
nit: maybe rephrase this comment ? not sure i'm parsing it well

Reply:
It's no longer accurate in this version, I'll rewrite it.